Jenkinsfile

```diff
@@ -423,14 +458,26 @@ node('py35') {
          credentialsId: '7155bd15-767d-4ae3-a375-e0d74c90a2c4',
          accessKeyVariable: 'AWS_ACCESS_KEY_ID',
          secretKeyVariable: 'AWS_SECRET_ACCESS_KEY'],
-        [$class: 'StringBinding',
-         credentialsId: 'fd1fe0ae-113d-4096-87b2-15aa9606bb4e',
-         variable: 'CF_TEMPLATE_URL'],
         [$class: 'UsernamePasswordMultiBinding',
          credentialsId: '323df884-742b-4099-b8b7-d764e5eb9674',
          usernameVariable: 'DCOS_ADMIN_USERNAME',
          passwordVariable: 'DCOS_ADMIN_PASSWORD']]) {
 
-        parallel builders
+        parallel binaryBuilders
+
+        def exceptions = []
+        for (version in DCOS_VERSIONS) {
```

Comment:
Can we reverse the order of execution. I think we should run against master first. Or even better run them all in parallel. Is that possible?